changes/ticket28113

```diff
@@ -0,0 +1,3 @@
+  o Minor features (relay usability):
+    - Notify systemd of ShutdownWaitLength so it can be set to more than 90
```

Comment:
The Tor default is actually 30s:
https://github.com/torproject/tor/blob/master/contrib/dist/tor.service.in
```suggestion
    - Notify systemd of ShutdownWaitLength so it can be set to more than 30
```

Reply:
Yes, but the systemd default is 90.